ui-tests/src/test/resources/features/integrations/sql-connector.feature

```diff
@@ -0,0 +1,64 @@
+# @sustainer: jsafarik@redhat.com
+
+@ui
+@database
+@log
+@sql-connector
+Feature: SQL Connector
+
+  Background: Clean application state
+    Given clean application state
+    And log into the Syndesis
+    And reset content of "todo" table
+    And navigate to the "Home" page
+  #
+  # 1. Check that SQL connector pulls all rows on exchange
+  #
+  @reproducer
+  @gh-3856
+  @sql-connector-all-rows
+  Scenario: Check SQL pulls all rows
+
+    Then inserts into "todo" table
+      | ABCsampleTODO |
+      | XYZsampleTODO |
+
+    When click on the "Create Integration" button to create a new integration
+    Then check visibility of visual integration editor
+    And check that position of connection to fill is "Start"
+
+    # select postgresDB as 'from' point
+    When select the "PostgresDB" connection
+    And select "Periodic SQL Invocation" integration action
+    Then check "Next" button is "Disabled"
+
+    # fill in postgresDB connection
+    When fill in periodic query input with "SELECT * FROM todo" value
+    And fill in period input with "1" value
+    And select "Minutes" from sql dropdown
+    And click on the "Next" button
+    Then check visibility of page "Choose a Finish Connection"
+
+    # select Log as 'to' point
+    When select the "Log" connection
+    And select "Simple Logger" integration action
+    And fill in values
+      | log level      | INFO  |
+      | Log Body       | true  |
+      | Log message Id | false |
+      | Log Headers    | false |
+      | Log everything | false |
+
+    Then click on the "Done" button
+
+    # save the integration with new name and publish it
+    When click on the "Save" button
+    And set integration name "DB_2_Log"
+    And publish integration
+    Then Integration "DB_2_Log" is present in integrations list
+    And wait until integration "DB_2_Log" gets into "Running" state
+
+    # validate that both items from db are present
+    When sleep for "10000" ms
+    Then .*validate that logs of integration "DB_2_Log" contains string "ABCsampleTODO"
```

Comment:
I believe this would pass even before - the problem happens when the SQL connection is in the middle of an integration